core-blocks/heading/index.js

```diff
@@ -1,16 +1,60 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
 import { concatChildren } from '@wordpress/element';
-import { createBlock, getPhrasingContentSchema } from '@wordpress/blocks';
+import {
+	createBlock,
+	getPhrasingContentSchema,
+	getBlockAttributes,
+	getBlockType,
+} from '@wordpress/blocks';
 import { RichText } from '@wordpress/editor';
 
 /**
  * Internal dependencies
  */
 import edit from './edit';
 
+/**
+ * Given a node name string for a heading node, returns its numeric level.
+ *
+ * @param {string} nodeName Heading node name.
+ *
+ * @return {number} Heading level.
+ */
+export function getLevelFromHeadingNodeName( nodeName ) {
+	return Number( nodeName.substr( 1 ) );
+}
+
+const supports = {
+	className: false,
+	anchor: true,
+};
+
+const schema = {
```

Comment:
We sometimes call this schema and sometimes attributes. We should be consistent.